app/components/primer/alpha/action_menu/action_menu_element.ts

```diff
@@ -261,7 +261,7 @@ export class ActionMenuElement extends HTMLElement {
       this.#hide()
     } else {
       this.#show()
-      this.#firstItem?.focus()
+      setTimeout(() => this.#firstItem?.focus(), 10)
```

Comment:
`this.#show()` just calls `showPopover` and in the related [docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/showPopover) it says

> When showPopover() is called on an element with the [popover](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover) attribute that is currently hidden, a [beforetoggle](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforetoggle_event) event will be fired, followed by the popover showing, and then the [toggle](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/toggle_event) event firing. If the element is already showing, an error will be thrown.

That sounds like we could wait for the `toggle` event, no? But I'm not sure as that would depend on not just if the popover is shown but also that the position has been set.